m/melky.md

```diff
@@ -0,0 +1,9 @@
+# Melky
+
+![Melky](melky.JPG)
+Her real name is Randrianarison Melky.
```

Comment:
New line between the image and paragraph please.

Reply:
I have put new line between the image and paragraph.